picasso/src/main/java/com/squareup/picasso3/RequestHandler.java

```diff
@@ -125,7 +125,7 @@ public boolean hasDrawable() {
      * {@link #load(Picasso, Request, Callback)} call. This is only accessible to built-in
      * RequestHandlers.
      */
-    int getExifOrientation() {
+    public int getExifOrientation() {
```

Comment:
We might want to rename this EXIF _rotation_. Something to consider.

Reply:
I'll update this in the next PR, since renaming will touch files which have nothing to do with this PR.

Reply:
Updated.